Resources/doc/libraries-versions.md

```diff
@@ -0,0 +1,14 @@
+JavaScript libraries' versions
+==============
+
+You might want to use custom versions of the libraries included by GraphiQL.
+This can be achieved in the bundle configuration, example:
+
+```yaml
+overblog_graphiql:
+    javascript_libraries:
+        graphiql: "0.11"
+        react: "15.6"
+        fetch: "2.0"
+        relay: "classic"
```

Comment:
I think this config is useless in this project, it was introduce in GraphQLBundle to switch between modern an classic relay json format when using command dump file

Reply:
Yep, done. and there's a new PR removing it from the project